addons/miscFixes/patchPLA/config.cpp

```diff
@@ -0,0 +1,593 @@
+#include "\z\potato\addons\miscFixes\script_component.hpp"
+#undef COMPONENT
+#define COMPONENT miscFixes_patchPLA
+
+class CfgPatches {
+    class ADDON {
+        weapons[] = {};
+        units[] = {"O_ZBL09", "O_ZTL11", "O_ZBD04A", "O_ZBD05","O_ZTD05"};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"ZBL09_LK", "ZTL11_LK", "ZBD04_LK", "ZBD05_LK", "ZTD05_LK"};
+        skipWhenMissingDependencies = 1;
+        author = "Potato";
+        authors[] = {"Chesheire"};
+        authorUrl = "https://github.com/BourbonWarfare/POTATO";
+        VERSION_CONFIG;
+    };
+};
+
+class CfgVehicles {
+	class Car;
+	class Car_F: Car {
+		class HitPoints {
+			class HitBody;
+			class HitEngine;
+			class HitFuel;
+			class HitHull;
+			class HitLFWheel;
+			class HitLBWheel;
+			class HitLMWheel;
+			class HitLF2Wheel;
+			class HitRFWheel;
+			class HitRBWheel;
+			class HitRMWheel;
+			class HitRF2Wheel;
+		};
+	};
+	class Wheeled_APC_F: Car_F {
+		class NewTurret;
+		class Turrets {
+			class MainTurret: NewTurret {
+				class ViewOptics;
+				class ViewGunner;
+				class Turrets {
+					class CommanderOptics;
+				};
+			};
+		};
+	};
+    class LandVehicle;
+	class Tank: LandVehicle {
+		class NewTurret;
+		class HitPoints;
+	};
+    class Tank_F: Tank {
+		class Turrets {
+			class MainTurret: NewTurret {
+				class Turrets {
+					class CommanderOptics;
+				};
+			};
+		};
+		class HitPoints: HitPoints {
+			class HitHull;
+			class HitEngine;
+			class HitLTrack;
+			class HitRTrack;
+			class HitFuel;
+		};
+	};
+	class ZBL09_base: Wheeled_APC_F {
```

Comment:
what are we basing this on?

Reply:
All values were normalized against the config pulled from the latest config found in this thread: https://forums.bohemia.net/forums/topic/191737-updated-all-in-one-config-dumps/?page=2

I couldn't find anything more recent than that, and when I tried to do my own pull the values were the same. In testing, it seemed to work in local as well :)

Reply:
latest _strictly speaking_ was published in the ACE discord: https://discord.com/channels/976165959041679380/976224730422063214/1148641265794154616

but what do you mean normalized against them? what was changed?

Reply:
discussed in voice: all were based on vanilla CSAT vehicles